### ISSUE TITLE
Require without import should issue an error when --noimplicitany is set

### ISSUE THREAD

**Issue by NoelAbrahams** [Microsoft/TypeScript#1569]
Hi,

**TS**: 1.3

The following compiles without error (with `noimplicitany` set):

```TypeScript
var foo = require('foo');
foo.bar = 10; // works because foo is implicitly typed as 'any'
```

The compiler should issue an error here because it permits `any` to leak into code files.

**Comment by danquirk** [comment]
Sounds like things are working as intended on the TypeScript side then.
